frontend/components/menu/MenuSearchResult.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div
     @mouseleave="showTooltip = false"
+    @blur="showTooltip = false"
```

Comment:
I'm gonna revert these changes as it's actually not the desired behavior :) This div is to add the `p-2` around the button or the tooltip if it's shown and allow for the mouse leave only, but shouldn't be focused itself. The focus should go directly to the button. If we need these in here for a11y reasons, then let's discuss it in a sync as this component is actually pretty fine tuned for usability 😊